docs/conferences/dayofevent/README.md

```diff
@@ -47,6 +47,11 @@ In order to provide a better stream to the attendees, based on their proximate d
 ```
 NEXT_PUBLIC_IPINFO_TOKEN="token from ipinfo"
 ```
+
+If you are using __Twitch__ RTMP URI then in order to view the stream you need to add specify your Twitch username in the `src` prop of the Videostream tag in the Mainstage component `/app/components/conferences/dayOfEvent/mainstage/Mainstage.js`.
+
```

Comment:
Nice, you wrote the documentation. 

One addition is required could you please also add more detailed documentation of this `VideoStreamer` component in `https://github.com/RocketChat/RC4Conferences/tree/main/docs/components`? Since you have been working on this feature, so you are definitely much more familiar with it.

Use [this](https://github.com/RocketChat/RC4Conferences/wiki/Event-Management-Components#event-scheduler) component as a reference; once you have done this, I will copy and those to our wiki page. Oh, and for the "Preview" part, please feel free to add a video.

Thank you!

Reply:
Hello @Dnouv 
I have added the wiki documentation of the videostream component, Since this is my first time writing documentation, kindly review it and point out any mistakes I've made. I've included the wiki documentation for the videostream component to the repository at `/docs/components/videostreamer/README.md`.

Reply:
Thank you! I will read it, from what I have read till now, it is written pretty well, nice!